front-end-app/src/components/ModalSimpleForm.js

```diff
@@ -9,34 +9,27 @@ const inlineStyle = {
     marginRight: 'centered'
   }
 };
-class ModalSimpleForm extends React.Component {
-
-
-  render () {
-    const form = this.props.form;
-
+const ModalSimpleForm = (props) =>{
     return(
-
       <Modal
         size='large'
         dimmer='inverted'
         style = {inlineStyle.modal}
-        open = {this.props.isOpen}
+        open = {props.isOpen}
```

Comment:
Then the props can just be:
```suggestion
        open = {isOpen}
```

and so on...

Reply:
yup, much cleaner, I'll continue to make this change.